src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeRevurderFraValideringTest.kt

```diff
@@ -0,0 +1,204 @@
+                    behandling,
+                    opprettVilkårperiodeMålgruppe(fom = revurderFra.minusDays(1)),
+                )
+                validerNyPeriodeIForholdTilRevurderFra(
+                    behandling,
+                    opprettVilkårperiodeMålgruppe(fom = revurderFra.plusDays(1)),
+                )
+            }
+        }
+
+        @Test
+        fun `kan legge inn periode som begynner samme eller etter revurderingsdato`() {
+            assertDoesNotThrow {
+                validerNyPeriodeIForholdTilRevurderFra(
+                    behandlingMedRevurderFra,
+                    opprettVilkårperiodeMålgruppe(fom = revurderFra),
+                )
+                validerNyPeriodeIForholdTilRevurderFra(
+                    behandlingMedRevurderFra,
+                    opprettVilkårperiodeMålgruppe(fom = revurderFra.plusDays(1)),
+                )
+            }
+        }
+
+        @Test
+        fun `kan ikke legge inn ny periode som begynner før revurder-fra`() {
+            assertThatThrownBy {
+                val vilkårperiode = opprettVilkårperiodeMålgruppe(fom = revurderFra.minusDays(1))
+                validerNyPeriodeIForholdTilRevurderFra(behandlingMedRevurderFra, vilkårperiode)
+            }.hasMessageContaining("Kan ikke opprette periode")
+        }
+    }
+
+    @Nested
+    inner class SlettPeriode {
+
+        @Test
+        fun `kan slette periode med valgfritt dato dersom revurder-fra ikke er satt`() {
+            assertDoesNotThrow {
+                validerSlettPeriodeIForholdTilRevurderFra(
+                    behandling,
+                    målgruppe(fom = revurderFra.minusDays(1)),
+                )
+                validerSlettPeriodeIForholdTilRevurderFra(
+                    behandling,
+                    målgruppe(fom = revurderFra.plusDays(1)),
+                )
+            }
+        }
+
+        @Test
+        fun `kan slette periode som begynner samme eller etter revurderingsdato`() {
+            assertDoesNotThrow {
+                validerSlettPeriodeIForholdTilRevurderFra(
+                    behandlingMedRevurderFra,
+                    målgruppe(fom = revurderFra),
+                )
+                validerSlettPeriodeIForholdTilRevurderFra(
+                    behandlingMedRevurderFra,
+                    målgruppe(fom = revurderFra.plusDays(1)),
+                )
+            }
+        }
+
+        @Test
+        fun `kan ikke slette periode som begynner før revurder-fra`() {
+            assertThatThrownBy {
+                val vilkårperiode = målgruppe(fom = revurderFra.minusDays(1))
+                validerSlettPeriodeIForholdTilRevurderFra(behandlingMedRevurderFra, vilkårperiode)
+            }.hasMessageContaining("Kan ikke slette periode")
+        }
+    }
+
+    @Nested
+    inner class OppdateringAvPeriode {
+
+        @Test
+        fun `kan oppdatere periode med valgfritt dato dersom revurder-fra ikke er satt`() {
```

Comment:
Navnet på testen virker som om man skal kunne endre datoer. Kan evt. legge på ", uavhengig av datoene på perioden" på slutten om man vil ha med noe om datoer
```suggestion
        fun `kan oppdatere periode dersom revurder-fra ikke er satt`() {
```